src/aco/ant.rs

```diff
@@ -17,6 +17,33 @@ pub trait Ant {
   fn go_to_next_place(&mut self, edges_goodness: &FMatrix) -> (usize, usize);
 }
 
+macro_rules! standard_ant_impl {
+  () => {
+    /// Clears iteration specific data like visited vertices or path.
+    fn clear(&mut self) {
+      for i in 0..self.solution_size {
+        self.unvisited.push(i);
+      }
+      self.path.clear();
+      self.stuck = false;
+    }
+    /// Returns vector of vertices in order of visiting
+    fn get_path(&self) -> &[usize] {
```

Comment:
Just a naming notice: 

[it is preferred not to use `get_` prefix](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter). Just replace it with:

```suggestion
    fn path(&self) -> &[usize] {
```